internal/proxyservice/http/aws/aws.go

```diff
@@ -20,9 +20,14 @@ import (
 // From https://github.com/aws/aws-sdk-go/blob/master/aws/signer/v4/v4.go#L77
 const timeFormat = "20060102T150405Z"
 
-// AWS4-HMAC-SHA256 Credential=AKIAJC5FABNOFVBKRWHA/20171103/us-east-1/ec2/aws4_request
+// reForCredentialComponent matches headers strings like:
+//
+//     AWS4-HMAC-SHA256 Credential=AKIAJC5FABNOFVBKRWHA/20171103/us-east-1/ec2/aws4_request
```

Comment:
The `AWS4-HMAC-SHA256` would have been stripped off by the time the matcher is run.
```suggestion
//     Credential=AKIAJC5FABNOFVBKRWHA/20171103/us-east-1/ec2/aws4_request
```